Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml

```diff
@@ -222,13 +222,16 @@
     </graphicData>
     <equippedAngleOffset>-150</equippedAngleOffset>
     <rotatable>false</rotatable>
-    <stackLimit>25</stackLimit>
```

Comment:
More beer! Yes, always good to have more beer per stack. lol
Even if it's only one more, it's still more beer! I always approve of more beer